### DISCUSSION TITLE
ERRO NO DOMINIO DO MEU SITE

### DISCUSSION THREAD

**Discussion by HeitorACB** [vercel/community#4077] [Help]
### Question

TENHO UM DOMINIO JA COMPRADO, POREM NO GOOGLE QUANDO PESQUISA MINHA EMPRESA ELE DA DOIS LINKS QUE NAO EXISTEM MAIS, E EU PRECISAVA FAZER O REDIRECIONAMENTO DESSES LINKS PARA O MEU SITE ATUAL, POREM A VERCEL NAO DEIXA EU COLOCAR O DOMINIO /EMPRESAS.PHP SENDO QUE MEU SITE ERRADO É .../EMPRESAS.PHP E PRECISO QUE ESTE DOMINIO LEVE PARA O MEU SITE ORIGINAL

**Comment by amyegan** [comment]
Hi @HeitorACB. If you own the domains that appear on your Google search, then you can point them to your site by changing the DNS records or setting up [Redirects](https://vercel.com/docs/edge-network/redirects#). If you do not own those domains, then it is not possible to redirect them to your site.
